src/UI/AddSeries/Existing/AddExistingSeriesCollectionView.js

```diff
@@ -30,9 +30,12 @@ module.exports = Marionette.CompositeView.extend({
 
         if (model) {
             var currentIndex = index;
-            var folderName = model.get('folder').name;
+            var termSearch = model.get('folder').name;
+            if (typeof model.get('folder').id !== "undefined") {
```

Comment:
`model.get('folder').id != null` (instead of the `!==` you had before as `!=` will catch undefined and null.